app/src/test/java/com/myfitnesstracker/NutritionDataUnitTest.kt

```diff
@@ -22,18 +22,21 @@ class NutritionDataUnitTest {
 
 
     private fun givenAFeedOfNutritionDataAreAvailable() {
-        mvn = MainViewModel()
+        //Changed from mvn to mvm
```

Comment:
Good, but remove discussion from code.  Keep that in the pull request comments.  It's clutter when it is in code.